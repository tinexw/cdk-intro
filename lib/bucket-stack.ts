import * as cdk from '@aws-cdk/core';

export class BucketStack extends cdk.Stack {
  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
    super(scope, id, props);
  }
}