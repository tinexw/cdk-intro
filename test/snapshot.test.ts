import { SynthUtils } from '@aws-cdk/assert';
import { Stack } from '@aws-cdk/core';

import { BucketStack } from '../lib/bucket-stack';

test('bucket stack', () => {
  const stack = new Stack();
  new BucketStack(stack, 'TestBucketStack');
  expect(SynthUtils.toCloudFormation(stack)).toMatchSnapshot();
});