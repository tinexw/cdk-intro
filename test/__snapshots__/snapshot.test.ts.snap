// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bucket stack 1`] = `
Object {
  "Resources": Object {
    "Bucket83908E77": Object {
      "DeletionPolicy": "Delete",
      "Properties": Object {
        "BucketName": "my-bucket-foo",
        "Tags": Array [
          Object {
            "Key": "environment",
            "Value": "foo",
          },
          Object {
            "Key": "foo",
            "Value": "bar",
          },
        ],
      },
      "Type": "AWS::S3::Bucket",
      "UpdateReplacePolicy": "Delete",
    },
  },
}
`;
